frontend/packages/operator-lifecycle-manager/src/components/create-operand.tsx

```diff
@@ -506,10 +746,8 @@ export const CreateOperandForm: React.FC<CreateOperandFormProps> = ({
             <ResourceRequirements
               cpu={_.get(formValues, [field.path, 'requests', 'cpu'])}
               memory={_.get(formValues, [field.path, 'requests', 'memory'])}
-              onChangeCPU={(cpu) => updateFormValues([field.path, 'requests', 'cpu'], cpu)}
-              onChangeMemory={(memory) =>
-                updateFormValues([field.path, 'requests', 'memory'], memory)
-              }
+              onChangeCPU={(cpu) => updateFormValues(`${field.path}.requests.cpu`, cpu)}
```

Comment:
Changed this so that it creates flat form values like everything else. Before, it would create a nested object in the formValues state, which created an unnecessary edge case.